pkg/controller/nodes/task/config/config.go

```diff
@@ -40,7 +39,7 @@ type Config struct {
 	MaxPluginPhaseVersions int32            `json:"max-plugin-phase-versions" pflag:",Maximum number of plugin phase versions allowed for one phase."`
 	BarrierConfig          BarrierConfig    `json:"barrier" pflag:",Config for Barrier implementation"`
 	BackOffConfig          BackOffConfig    `json:"backoff" pflag:",Config for Exponential BackOff implementation"`
-	MaxErrorMessageLength  int              `json:"maxLogMessageLength" pflag:",Max length of error message."`
+	MaxErrorMessageLength  int              `json:"maxLogMessageLength" pflag:",Deprecated!!! Max length of error message."`
```

Comment:
Do we have this config in two places? is that why? can you add that to the deprecation message? "Use Blah.Blah.MaxErrorMessageLength" ?

Reply:
no this is not used.